tools/torchfix/torchfix/visitors/deprecated_symbols/__init__.py

```diff
@@ -89,7 +89,7 @@ def __init__(self):
     def leave_ImportFrom(
         self, node: cst.ImportFrom, updated_node: cst.ImportFrom
     ) -> cst.ImportFrom:
-        if node.module.value == "functorch" and all(
+        if getattr(node.module, "value", None)  == "functorch" and all(
             name.name.value in self.REPLACEMENTS for name in node.names
```

Comment:
Just do be on the safe side, don't you need the same here?
```suggestion
            getattr(name.name, "value", None) in self.REPLACEMENTS for name in node.names
```

Reply:
From https://docs.python.org/3/library/ast.html for `alias` always has name, so `name.name` is never None.